packages/clay-navigation/src/Breadcrumb.tsx

```diff
@@ -0,0 +1,122 @@
+/**
+ * © 2019 Liferay, Inc. <https://liferay.com>
+ *
+ * SPDX-License-Identifier: BSD-3-Clause
+ */
+
+import classNames from 'classnames';
+import React from 'react';
+import warning from 'warning';
+import {BreadcrumbEllipsis} from './BreadcrumbEllipsis';
+import {BreadcrumbItem, IBreadcrumbItem} from './BreadcrumbItem';
+
+interface IProps extends React.HTMLAttributes<HTMLOListElement> {
+	ellipsisBuffer?: number;
+
+	ellipsisProps?: Object;
+
+	items: Array<IBreadcrumbItem>;
+
+	spritemap: string;
+}
+
+const getBufferList = (
+	items: Array<IBreadcrumbItem>,
+	start: number,
+	end: number,
+	ellipsisProps: Object,
+	spritemap: string
+): (IBreadcrumbItem | React.ReactElement)[] => {
+	const removedItems = items.slice(start, Math.max(end, start));
+
+	return removedItems.length > 1
+		? [
+				<BreadcrumbEllipsis
+					items={removedItems}
+					key="breadcrumbEllipsis"
+					spritemap={spritemap}
+					{...ellipsisProps}
+				/>,
+		  ]
+		: removedItems;
+};
+
+const getBreadcrumbItems = (
+	ellipsisBuffer: number,
+	items: Array<IBreadcrumbItem>,
+	ellipsisProps: Object,
+	spritemap: string
+) => {
+	const activeIndex = items.findIndex(item => item.active);
```

Comment:
Im not sure this is supported in IE11. https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/findIndex#Browser_compatibility

Reply:
Replacing findIndex with a filter length.